doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -107,418 +140,261 @@ spec:
 EOF
 ```
 
-Then wait for the `ClusterIssuer` to become ready as follows:
+Wait for the `ClusterIssuer` to become ready.
 
 ```bash
-kubectl wait clusterissuer/${clusterIssuerName} --for=condition=ready=true
+kubectl wait clusterissuer/${KUADRANT_CLUSTER_ISSUER_NAME} --for=condition=ready=true
 ```
 
-### Step 4 - Set up a Gateway
-
-For Kuadrant to balance traffic using DNS across two or more clusters, you must define a Gateway with a shared host. You will define this by using an HTTPS listener with a wildcard hostname based on the root domain. As mentioned earlier, you must apply these resources to all clusters. 
-
-NOTE: For now, the Gateway is set to accept an `HTTPRoute` from the same namespace only. This allows you to restrict who can use the Gateway until it is ready for general use.
+### Deploy a Gateway
 
 ```bash
 kubectl apply -f - <<EOF
 apiVersion: gateway.networking.k8s.io/v1
 kind: Gateway
 metadata:
-  name: ${gatewayName}
-  namespace: ${gatewayNS}
+  name: ${KUADRANT_GATEWAY_NAME}
+  namespace: ${KUADRANT_GATEWAY_NS}
   labels:
     kuadrant.io/gateway: "true"
 spec:
     gatewayClassName: istio
     listeners:
     - allowedRoutes:
         namespaces:
-          from: Same
-      hostname: "*.${rootDomain}"
+          from: All 
+      hostname: "*.${KUADRANT_ZONE_ROOT_DOMAIN}"
```

Comment:
might not be the worst ides to specify a none wildcard here as it would allow a healthcheck then also

Reply:
cool ill add it in